### PR TITLE
Added Blinking Button

### PR DESCRIPTION
**Pull request by adrsh79086**
Issue:

### CODE REVIEW

**Review by dsnehasish74 — CHANGES_REQUESTED** [review]
It looks great but the animation is not working for yellow and white buttons . Can you please work on this ???

**Comment by shahednasser** [comment]
@adrsh79086 please next time instead of creating a new PR, commit the changes to the previous one.

File: src/components/_animated.less [hunk 1]
```diff
@@ -25,4 +25,9 @@
 @import "animated/_loading-btn";
 @import "animated/_spring.less";
 @import "animated/_papercurl.less";
+<<<<<<< HEAD
```
Comment (shahednasser) [inline src/components/_animated.less:28]: Remove this line

File: src/components/_animated.less [hunk 2]
```diff
@@ -25,4 +25,9 @@
...
+@import "animated/_vibrate.less";
 @import "animated/_atom.less";
+=======
```
Comment (shahednasser) [inline src/components/_animated.less:31]: Remove this line

File: src/components/_animated.less [hunk 3]
```diff
@@ -25,4 +25,9 @@
...
+@import "animated/_blinking.less";
+>>>>>>> 6aee9781f9f81a1a331d6cb64799ca29258b8434
```
Comment (shahednasser) [inline src/components/_animated.less:33]: Remove this line

**Review by shahednasser — CHANGES_REQUESTED** [review]
Please do the changes below then run `npm run compile` and commit the changes.

**Pull request closed without merging**